igcollect/logfile_values.py

```diff
@@ -276,12 +293,17 @@ def main():  # NOQA: C901
                         for line in fh:
                             get_metrics_values(line, args.metric,
                                                args.time_format,
-                                               args.columns_num)
+                                               args.columns_num,
+                                               args.time_column,)
 
     for metric in args.metric:
         template = args.prefix + '.{} {} ' + str(metric.now)
-        print(template.format(metric.name, metric.get_metric_value()))
-
+        value = metric.get_metric_value()
+        if isinstance(value, dict):
+            for k, v in value.items():
```

Comment:
A `dict` here can not have further `dict`s inside? Is it sure that there is no recursion?

Reply:
Did not get